Content.Shared/Throwing/ThrownItemSystem.cs

```diff
@@ -114,18 +119,25 @@ private void StopThrow(EntityUid uid, ThrownItemComponent thrownItemComponent)
                 }
             }
 
-            EntityManager.EventBus.RaiseLocalEvent(uid, new StopThrowEvent {User = thrownItemComponent.Thrower}, true);
+            var stopThrow = new StopThrowEvent(thrownItemComponent.Thrower);
+            EntityManager.EventBus.RaiseLocalEvent(uid, ref stopThrow, true);
```

Comment:
use  the proxy method RaiseLocalEvent() , do the same for the rest of things calling EntityManager in here